cli/exchange/org.go

```diff
@@ -55,13 +62,16 @@ func OrgList(org, userPwCreds, theOrg string, long bool) {
 	}
 }
 
-func OrgCreate(org, userPwCreds, theOrg string, label string, desc string, min int, max int, adjust int) {
+func OrgCreate(org, userPwCreds, theOrg string, label string, desc string, tags []string, min int, max int, adjust int, maxNodes int, agbots []string) {
```

Comment:
We dont support more than 1 agbot serving the same org.

Reply:
done